src/commands.ts

```diff
@@ -91,6 +91,30 @@ export function registerCommands(context: vscode.ExtensionContext, prManager: IP
 		});
 	}));
 
+	context.subscriptions.push(vscode.commands.registerCommand('pr.merge', async (pr?: PRNode, message?: string) => {
+		const pullRequest = ensurePR(prManager, pr);
+		return vscode.window.showWarningMessage(`Are you sure you want to merge this pull request on GitHub?`, 'Yes', 'No').then(async value => {
```

Comment:
From Peng's suggestion to use a modal instead of a notification, we can add `{ modal: true }` after the first argument

Reply:
Fixed by just add a your suggestion argument.